linkerd/app/integration/src/client.rs

```diff
@@ -93,6 +95,13 @@ pub struct Client {
     tls: Option<TlsConfig>,
 }
 
+pub struct NewClient {
```

Comment:
tioli: Clearer to call this `Reconnect`? having `shutdown(self) -> Reconnect` is a little clearer than `shutdown(self) -> NewClient`

Reply:
good call, renamed!